Dockerfile

```diff
@@ -7,7 +7,13 @@ WORKDIR /src
 # Copy project .dockerignore prevents copy of unnecessary files
 COPY . .
 
-# Gradle task build and extract dependencies with daemon disabled
+# Gradle task for linter to check errors with daemon disabled
+RUN ./gradlew ktlintCheck --no-daemon
```

Comment:
[Gradle documentation states this](https://docs.gradle.org/current/userguide/gradle_daemon.html)

> 
  | Continuous integration  Since Gradle 3.0, we enable Daemon by default and recommend using it  for both developers' machines and Continuous Integration servers.  However, if you suspect that Daemon makes your CI builds unstable, you  can disable it to use a fresh runtime for each build since the runtime  is completely isolated from any previous builds.
-- | --

Maybe this too pre-emptive and we should wait and see if this issue arises?